syncweek-react-aad/react-aad
# Expose the full AuthResponse details

## Purpose

Currently we only expose the `Account` info when authentication completes. This is not sufficient because, for example, this would not give access to the `state` data which is needed for many security scenarios. The full authentication details are exposed by `msal` using the `AuthResponse`, and we should be returning this data and storing in Redux.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Run the sample app and verify the `AuthResponse` is saved in the Redux store as `authenticationResponse` using the React DevTools extension.
